config.def.keybinds.h

```diff
@@ -153,7 +157,11 @@ static const struct retro_keybind retro_keybinds_1[] = {
    { true, RARCH_LOAD_STATE_KEY,           MENU_ENUM_LABEL_VALUE_INPUT_META_LOAD_STATE_KEY,        RETROK_F4,        NO_BTN, NO_BTN, 0, AXIS_NONE, AXIS_NONE, AXIS_NONE, NULL, NULL },
    { true, RARCH_SAVE_STATE_KEY,           MENU_ENUM_LABEL_VALUE_INPUT_META_SAVE_STATE_KEY,        RETROK_F2,        NO_BTN, NO_BTN, 0, AXIS_NONE, AXIS_NONE, AXIS_NONE, NULL, NULL },
    { true, RARCH_FULLSCREEN_TOGGLE_KEY,    MENU_ENUM_LABEL_VALUE_INPUT_META_FULLSCREEN_TOGGLE_KEY, RETROK_f,         NO_BTN, NO_BTN, 0, AXIS_NONE, AXIS_NONE, AXIS_NONE, NULL, NULL },
+#ifdef HAVE_LAKKA
+   { true, RARCH_QUIT_KEY,                 MENU_ENUM_LABEL_VALUE_INPUT_META_RESTART_KEY,           RETROK_ESCAPE,    NO_BTN, NO_BTN, 0, AXIS_NONE, AXIS_NONE, AXIS_NONE, NULL, NULL },
```

Comment:
This question is out of scope, but should we introduce a `RARCH_RESTART_KEY` to save having needing the `HAVE_LAKKA` conditional?

Reply:
Will be good to have.